linux_os/guide/services/kerberos/kerberos_disable_no_keytab/rule.yml

```diff
@@ -29,8 +29,7 @@ references:
     stigid@ol8: OL08-00-010161
     stigid@rhel8: RHEL-08-010161
 
-platforms:
-    - krb5_server_older_than_1_17-18 and krb5_workstation_older_than_1_17-18
+platform: package[krb5-server]<1.17-18 and package[krb5-workstation]<1.17-18
```

Comment:
@matejak Do you recommend to move the changes of real use of the feature (ie. changing platform to use versioned expressions) to a separate PR?

Reply:
@matejak ^